docs/Configuration.md

```diff
@@ -145,11 +145,85 @@ oc create secret generic snow-secret \
 -n pelorus
 ```
 
+### Labels
+
+Labels are key/value pairs that are attached to objects, such as pods. Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users and Pelorus.
+
+The commit time, deploy time, and failure exporters all rely on labels to indentify the application that is associated with an object.  The object can
+include a build, build configuration, deployment or issue.
+
+In Pelorus the required label is: `app.kubernetes.io/name=<app_name>` where 
+app_name is the name of the application(s) being monitored.  Each Pelorus exporter
+should be configured with an `APP_LABEL` or `APP_FIELD` with a comma delimited list
+of applications.
+
+> **Warning** 
+> If the application label is not properly configured per object it may result in Pelorus not displaying measurements in Grafana.  One can detect a misconfigured application label by ensuring the `DEBUG` log level is set in the
+exporter in question. 
+
+In the following examples an application named [todolist](https://github.com/konveyor/mig-demo-apps/blob/master/apps/todolist-mongo-go/mongo-persistent.yaml) is being monitored.
+
+Example BuildConfig:
+```
+ - kind: BuildConfig
+    apiVersion: build.openshift.io/v1
```

Comment:
Indentation issue on each of these examples. Also, we don't need the leading dash, right?

Reply:
ok.. fixed in next push.. thanks